network/p2p/libp2pNodeBuilder.go

```diff
@@ -0,0 +1,371 @@
+package p2p
```

Comment:
Spoiler Alert: no new logic in this file, just moved builders and defaults to this new file with very minor refactoring.